mycli/main.py

```diff
@@ -41,6 +41,8 @@
 from .lexer import MyCliLexer
 from .__init__ import __version__
 
+from mycli.encodingutils import text_type
```

Comment:
Looks like I told you wrong - you can add this to line 39 where we are already importing from `encodingutils` ;) Sorry about that!